tutorials/fiori-ios-scpms-floorplan/fiori-ios-scpms-floorplan.md

```diff
@@ -133,17 +142,28 @@ Implement the following lines of code directly below the loading indicator prope
 let destinations = FileConfigurationProvider("AppParameters").provideConfiguration().configuration["Destinations"] as! NSDictionary
 
 // Retrieve the data service using the destinations dictionary and return it.
-var dataService: ESPMContainer<OnlineODataProvider>? {
-    guard let odataController = OnboardingSessionManager.shared.onboardingSession?.odataControllers[destinations["com.sap.edm.sampleservice.v2"] as! String] as? Comsapedmsampleservicev2OnlineODataController, let dataService = odataController.espmContainer else {
-        AlertHelper.displayAlert(with: NSLocalizedString("OData service is not reachable, please onboard again.", comment: ""), error: nil, viewController: self)
+let destinations = FileConfigurationProvider("AppParameters").provideConfiguration().configuration["Destinations"] as! NSDictionary
+
+var dataService: ESPMContainer<OfflineODataProvider>? {
+    guard let odataController = OnboardingSessionManager.shared.onboardingSession?.odataControllers[ODataContainerType.eSPMContainer.description] as? ESPMContainerOfflineODataController, let dataService = odataController.dataService else {
+        AlertHelper.displayAlert(with: "OData service is not reachable, please onboard again.", error: nil, viewController: self)
         return nil
     }
     return dataService
 }
 
 ```
 
-> In case you're using an `ODataOfflineProvider` you have to change the above-mentioned code to use `ODataOfflineProvider` instead of `ODataOnlineProvider`. You have to also import `SAPOfflineOData` in addition to the `SAPOData` framework.
+
+```Swift
+review
+
+```
+
+
+
+
+> In case you're using an `OfflineODataProvider` you have to change the above-mentioned code to use `OfflineODataProvider` instead of `OnlineODataProvider`. You have to also import `SAPOfflineOData` in addition to the `SAPOData` framework.
 
 Because we're good citizens we want to use an app logger to log important information. Fortunately, SAP is offering a simple-to-use Logging API with the `SAPCommon` framework.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. When giving instructions to the user, say "you" instead. Please also check the rest ot the tutorial and make necessary changes.